_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Run The Test Suite Locally
+order: 045
+---
+
+# {{ page.title }}
+
+This page describes how to run the integration test suite for your Forseti
+contributions on your local dev environment. 
+
+## **Setting up development environment**
+
+- Create a new project in your organization.
+- Terraform uses an IAM Service Account to deploy and configure resources on 
+behalf of the user. You can create the Service account, and grant the necessary 
+roles to the Service Account manually or by running the helper script. 
+
+### **Creating Service Account by running the helper script**
+
+In order to execute this script, you must have an account with the
+`Billing Account Administrator` role should be granted to the admin account. This
+is required to create the Service Account using the helper script. You can revoke 
+this role after the service account is created.
+- To run the [helper script](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/helpers/setup-sa.sh), 
+clone the [Terraform Google Project factory repository](https://github.com/terraform-google-modules/terraform-google-project-factory)
+and run the following command:
+
+`./helpers/setup-sa.sh <ORGANIZATION_ID> <PROJECT_NAME> <BILLING_ACCOUNT>`
```

Comment:
It doesn't look like this script does the same as what's listed below. For example, it does not add the roles organizationAdmin, cloudkms.admin, cloudsql.admin, etc. The service account will also need the following roles to create and delete a custom role for the Forseti server SA: `iam.roles.create, iam.roles.delete`.

Reply:
Updated to run the script from the TF repo and grant missing roles.